api/core/v1alpha1/resource.go

```diff
@@ -56,6 +61,14 @@ func ObjectCountQuotaResourceNameFor(groupResource schema.GroupResource) Resourc
 	return ResourceName(ResourceCountNamespacePrefix + groupResource.Resource + "." + groupResource.Group)
 }
 
+func IsClassCountResource(resource ResourceName) bool {
+	return strings.HasPrefix(string(resource), ClassCountPrefix)
+}
+
+func ClassCountFor(className string) ResourceName {
```

Comment:
We should disambiguate between different types of classes (i.e. `MachineClass`, `VolumeClass`, `BucketClass`).
What I'd recommend here is introducing a first parameter e.g. `classType` of type `string` so that the resulting `ResourceName` would be something like `class/machine.my-class-name`